src/renderer/components/ft-toast/ft-toast.vue

```diff
@@ -7,13 +7,18 @@
       :class="{ closed: !toast.isOpen, open: toast.isOpen }"
       tabindex="0"
       role="status"
-      @click="performAction(toast.id)"
```

Comment:
I believe removing this will break the ability to copy error messages for error toasts